coursebuilder/modules/explorer/_static/components/pages/homepage.html

```diff
@@ -37,44 +37,50 @@ <h1 class="intro-title">
 
 
       <!--  Awards  -->
-      <div class="section awards" hidden>
+      <div class="section awards">
```

Comment:
@andela-iakande @andela-angene DRY. Maybe templates?

Reply:
@chegejames Please I need you to expatiate more on this. I am unable to see any repetition here.

Reply:
@andela-iakande Awards section now Partners for example appears on both home and certification pages.

Reply:
Thanks for pointing this out @chegejames, a template have been used for that section.